examples/express-graphql-http-functions/Query.ts

```diff
@@ -0,0 +1,73 @@
+import { Context } from "./context";
+import User from "./models/User";
+
+/**
+ * @gqlType
+ */
+export type Query = unknown;
```

Comment:
Our integration tests currently assume all examples implement a core set of schema and will answer to a set of common queries: https://github.com/captbaritone/grats/blob/6753bc27e316dc87ff36e456809a65e46833c903/src/tests/integration.mjs#L23

We could either expand this example to also implement that part of the scheme, or (perhaps better) we could come up with a convention where example projects define a set of operations/expected responses that are specific to the example and test the unique features it offers.